src/core/scene/a-scene.js

```diff
@@ -294,9 +296,15 @@ class AScene extends AEntity {
         var xrMode = useAR ? 'immersive-ar' : 'immersive-vr';
         xrInit = this.sceneEl.systems.webxr.sessionConfiguration;
         return new Promise(function (resolve, reject) {
-          navigator.xr.requestSession(xrMode, xrInit).then(
+          var requestFunction = useOfferSession ? navigator.xr.offerSession.bind(navigator.xr) : navigator.xr.requestSession.bind(navigator.xr);
```

Comment:
can just call `requestSession`

Reply:
no, we need to call offerSession here to show the button

Reply:
Sorry, I meant renaming `requestFunction` variable to `requestSession`